docs/AppendixA-Troubleshooting_guideline.md

```diff
@@ -38,7 +38,7 @@ _**Table 3: Common errors encountered during execution of the LORIS-MRi insertio
 |`... error message = 'No space left on device ...'`|The temporary directory where the insertion scripts peform its intermediate steps is full. This directory is set to a default value of `/tmp` as specified in the line `export TMPDIR=/tmp` of the `environment` file |Change the `TMPDIR` path in the environment file to a directory with enough space. A good rule of thumb is to have at least 2-3 times the size of the scan being processed as writable space. This space is usually automatically emptied by the pipeline upon a successful execution|
 |`ERROR: This class is designed around the notion of a 'Study'. You can't use it with data from multiple studies.  The following study UIDs were found: '1.3.12.2.1107.66060.300000004' '1.3.12.2.1107.66060.300000007' The dicomTar execution has failed`|The upload contains acquisitions from two scanning sessions|Separate into two separate uploads|
 |`Out of memory!` during the execution of `dicomTar.pl`|The Transfer syntax of the uploaded scan is other than Little Endian Explicit (such as Implicit or JPEG lossless)|Use the DICOM toolkit `dcmconv` to convert. An example command that changes the transfer syntax to Little Endian Explicit on all DICOM files within a given folder is: `for i in $(find -type f); do dcmconv --write-xfer-little $i $i; done`|
-|`Number of MINC files that will be considered for inserting into the database: 0. No data could be converted into valid MINC files. Localizers will not be considered!`|The upload contains only ‘localizer’ type SeriesDescription|Localizers are not processed by default in LORIS|
+|`Number of MINC files that will be considered for inserting into the database: 0. No data could be converted into valid MINC files. Your project's excluded series descriptions will not be considered!`|The upload contains only acquisitions that have SeriesDescriptions which are excluded by the project|Projects can specify what acquisitions' SeriesDescription are not to be processed by default in LORIS|
```

Comment:
maybe remove `your project excluded series description` from the Code format? since now you made it such that every project will display something different here based on how they configured it...